bruno/collections/Rafiki/scripts.js

```diff
@@ -1,11 +1,16 @@
+const { createHmac } = require('crypto')
+const { canonicalize } = require('json-canonicalize')
 const fetch = require('node-fetch')
 const url = require('url')
 
 const scripts = {
   sanitizeUrl: function () {
     return req
       .getUrl()
-      .replace(/{{([A-Za-z]\w+)}}/g, (_, key) => bru.getEnvVar(key))
+      .replace(
+        /{{([A-Za-z]\w+)}}/g,
+        (_, key) => bru.getVar(key) || bru.getEnvVar(key)
```

Comment:
Bruno Collection variables take precedence over Bruno Environment Variables, [per the docs.](https://docs.usebruno.com/scripting/javascript-reference#collection-variables)

Reply:
But we don't have collection variables, do we?

Reply:
I didn't check _every_ API request for this, but _Deposit Asset Liquidity_ definitely does with `transferId` and `idempotencyKey`.